cli/azd/pkg/account/subscriptions.go

```diff
@@ -21,21 +22,25 @@ type SubscriptionsService struct {
 	credentialProvider auth.MultiTenantCredentialProvider
 	userAgent          string
 	httpClient         httputil.HttpClient
+	cloud              *cloud.Cloud
 }
 
 func NewSubscriptionsService(
 	credentialProvider auth.MultiTenantCredentialProvider,
-	httpClient httputil.HttpClient) *SubscriptionsService {
+	httpClient httputil.HttpClient,
+	cloud *cloud.Cloud,
+) *SubscriptionsService {
```

Comment:
Is there any way we can avoid propagating this up through our exposed service?  Maybe there is an opportunity to inject the `*armsubscriptions.Client` here instead and leave the internal cloud configuration to the IoC container.